vaadinfrontend/src/main/java/life/qbic/datamanager/views/AppRoutes.java

```diff
@@ -1,24 +1,44 @@
 package life.qbic.datamanager.views;
 
 /**
- * Defines the data manager's public application routes, that
- * shall be visible by all components within the app.
- *
+ * Defines the data manager's public application routes, that shall be visible by all components
+ * within the app.
+ * <p>
  * Local routes of components should be listed here with care!
  *
  * @since 1.0.0
  */
 public class AppRoutes {
 
   /**
-   * The "Main" page of the data manager app. Location, where you want the user to be
-   * directed to, if no explicit path has been provided.
+   * The "Main" page of the data manager app. Location, where you want the user to be directed to,
+   * if no explicit path has been provided. Shows all the projects a user has access to.
    */
-  public static final String MAIN = "";
+  public static final String PROJECTS = "";
```

Comment:
I'd argue that MAIN, HOME or ROOT was actually correct here since it's intention is to be the route for directing the user to the current landing page.